scannerlib/configchecks/content_entry_file_check_test.go

```diff
@@ -272,7 +272,9 @@ func TestFileContentEntryCheckComplianceResults(t *testing.T) {
 				t.Fatalf("Created %d checks, expected only 1", len(checks))
 			}
 
-			resultMap, err := checks[0].Exec()
+			var pVal string
+
+			resultMap, _, err := checks[0].Exec(pVal)
```

Comment:
Why the var declaration? Can't we just do checks[0].exec("")? Or with a random hardcoded value instead of an empty string.

Reply:
Agree, moved.